typescript/nomad-deploy/src/proxyUtils.ts

```diff
@@ -41,6 +41,12 @@ export type ProxyAddresses = {
   beacon: string;
 };
 
+export async function deployImplementation<T extends ethers.Contract>(deploy: Deploy, factory: ethers.ContractFactory, ...deployArgs: any[]): Promise<T> {
```

Comment:
this function shouldn't exist. it's just recreating functionality on the factory

Reply:
https://github.com/nomad-xyz/nomad-monorepo/pull/79/commits/6bb43d4df2b79ed9e8d0904e207e7e6f1393d776